Automattic/jetpack
# Donations: add custom icon

The Donations block icon is not extended with the paid symbol because it uses a Gridicon icon. A quick fix for this issue is simply to create a custom icon for this block.

#### Changes proposed in this Pull Request:

* Add custom icon for Donations block

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Apply D51974-code to test in Simple sites
* Select a site without a paid plan
* Go to create/edit a post
* Add a Donations block
* Confirm that now you see the paid symbol (pink ring) in the donations icon.
![image](https://user-images.githubusercontent.com/77539/97464042-2deb2d80-191f-11eb-9b8b-2e904a673c66.png)

* Confirm the paid symbols isn't present anymore for a site with a paid plan

![image](https://user-images.githubusercontent.com/77539/97464153-4f4c1980-191f-11eb-960a-beb35d11df06.png)


#### Proposed changelog entry for your changes:

*
